src/components/AppContainer.js

```diff
@@ -0,0 +1,111 @@
+import React, { Component } from 'react';
+import _ from 'lodash';
+import moment from 'moment';
+
+import Repeat from './Repeat/index';
+import End from './End/index';
+import RRule from './RRule/index';
+import computeRRule from '../utils/computeRRule';
+import { DATE_TIME_FORMAT } from '../constants/index';
+
+class AppContainer extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
```

Comment:
Hmm, when trying to move it outside the constructor I got the following error when I hover 'state':
`Conflicting inherited declaration React.Component.state was found in namespace public`

Reply:
There is and eslint error (unexpected token =) but this is actually working. And might be the issue with eslint itselft https://github.com/babel/babel-eslint/issues/312
This is stylistic suggestion so there is no need to change that.

Reply:
In emoji_roulette we have React 15.6.1 while here you have 16.0.0. This might be the case that WebStorm does not know where to find `import { Component }`

Reply:
According to state - it's webstorm related (not ESLint error)